src/rtctools/data/timeseries.py

```diff
@@ -0,0 +1,88 @@
+"""Module for processing timeseries data."""
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import numpy as np
+
+import rtctools.data.csv as csv
+import rtctools.data.pi as pi
+import rtctools.data.rtc as rtc
+
+
+def get_timeseries_from_csv(
+    file: Path, delimiter=","
+) -> Tuple[List[datetime], Dict[str, np.ndarray]]:
```

Comment:
This method is specific to CSV files and so should be in 'CSVMixin'

Reply:
I'll move it.